docs/sources/release-notes/next.md

```diff
@@ -13,3 +13,6 @@ Grafana Labs is excited to announce the release of Loki ?.?. Here's a summary of
 ## Features and enhancements
 
 -  **Query Language Improvements**: Several improvements to the query language that speed up line parsing and regex matching. [PR #8646](https://github.com/grafana/loki/pull/8646), [PR #8659](https://github.com/grafana/loki/pull/8659), [PR #8724](https://github.com/grafana/loki/pull/8724), [PR #8734](https://github.com/grafana/loki/pull/8734), [PR #8739](https://github.com/grafana/loki/pull/8739), [PR #8763](https://github.com/grafana/loki/pull/8763), [PR #8890](https://github.com/grafana/loki/pull/8890),[PR #8914](https://github.com/grafana/loki/pull/8914)
+
+-  **Multi-store Index support**: Loki now supports reading/writing indexes to multiple object stores which enables the use of different storage buckets across periods for storing index. [PR #7754](https://github.com/grafana/loki/pull/7754)], [PR #7447](https://github.com/grafana/loki/pull/7447)

```

Comment:
```suggestion
-  **Multi-store support**: Loki now supports reading/writing data to different object stores from the same cloud provider, enabling different storage buckets across periods for storing data. [PR #7946](https://github.com/grafana/loki/pull/7946)], [PR #7754](https://github.com/grafana/loki/pull/7754)], [PR #7447](https://github.com/grafana/loki/pull/7447)
```

Reply:
2.8.x already includes #7946, so being able to configure different object stores from the same cloud provider is already possible.

but the index shipper support to be able to sync indexes from multiple buckets (be it the same or different cloud provider) is added with 2.9.x. I tried to explicitly mention "index" as we have been able to write/read chunks across multiple buckets even before, we only had that limitation with the index

Reply:
Ahh okay. Sounds good then!

Reply:
docs are hard :dizzy_face: